apps/server/src/modules/deletion-console/deletion-client/deletion.client.ts

```diff
@@ -26,9 +26,9 @@ export class DeletionClient {
 		}
 	}
 
-	public async executeDeletions(limit?: number): Promise<void> {
+	public async executeDeletions(limit?: number, runFailed?: boolean): Promise<void> {
```

Comment:
Would it make sense to use `options: TriggerDeletionExecutionOptions` instead of sereate parameters? Would better show the independence of `limit` and `runFailed` IMHO. (maybe also a matter of taste)